src/components/shortcuts.js

```diff
@@ -177,6 +178,10 @@ function executeAction(card, target, action) {
 
     const item = getItemInfoFromCard(card);
 
+    const itemsContainer = dom.parentWithClass(card, 'itemsContainer');
+
+    const sortParentId = 'items-' + (item.IsFolder ? item.Id : itemsContainer.getAttribute('data-parentid')) + '-Folder';
```

Comment:
`itemsContainer` can be null - LiveTV Guide.
```suggestion
    const sortParentId = 'items-' + (item.IsFolder ? item.Id : itemsContainer?.getAttribute('data-parentid')) + '-Folder';
```